iOS/MobileStarter/react-app/src/HubScreenButton.tsx

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import classnames from "classnames";
+import { LoadingSpinner } from "@itwin/core-react";
+import { Button, ButtonProps } from "./Exports";
+
+export function HubScreenButton(props: ButtonProps) {
+  const { className, ...others } = props;
+  return <Button className={classnames("imodel-row", className)} {...others} />;
+}
+
+export interface HubScreenButtonListProps {
+  children?: React.ReactNode;
+  loading?: boolean;
+}
+
+export function HubScreenButtonList(props: HubScreenButtonListProps) {
+  const { children, loading } = props;
+  if (loading)
+    return <div className="centered"><LoadingSpinner /></div>;
+
+  return <div className="list">
```

Comment:
Similarly to HubScreenButton: with this being its own custom control, I don't think "list" is a good classname.

Reply:
Changed to hubscreen-list